polar-core/src/terms.rs

```diff
@@ -207,10 +212,11 @@ pub enum Value {
     Variable(Symbol),
     RestVariable(Symbol),
     Expression(Operation),
+    Partial(Constraints),
```

Comment:
Do we want to add a way to create these from polar? (other than I guess hacking it and registering the python Partial type as a class)

Reply:
I want to add the syntax `∂x` to mean `Partial(Variable(x))`, but maybe that's going to far.